variants/backend-base/.node-version

```diff
@@ -0,0 +1 @@
+16.14.0
```

Comment:
I know 16.15.0 is out now but it's not in `nodenv` yet so I couldn't use it today.